CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing Guidelines
+
+Contributions are welcome via [GitHub Pull Requests](https://docs.github.com/articles/about-pull-requests). This document outlines the process to help get your contribution accepted.
+
+Any type of contribution is welcome; from new features, bug fixes, or documentation improvements. However, [Engity](https://engity.com) will review the proposals and perform a triage over them. By doing so, we will ensure that the most valuable contributions for the community will be implemented in due time.
```

Comment:
Any kind of contribution is welcome, from new features to bug fixes to documentation improvements. However, [Engity](https://engity.com) will review the proposals and perform a triage over them. By doing so, we will ensure that the most valuable contributions for the community will be implemented in due time.